config/locales/en.yml

```diff
@@ -646,6 +646,13 @@ en:
     select_type:
       heading: Choose correspondence type
       sub_heading: Create case
+    show:
+      case/ico:
+        attachments_heading: Appeal response
+      case/sar:
```

Comment:
I think for this part we don't need to repeat Response for case/foi and case/sar I think if you do `    show:
      case/ico:
        attachments_heading: Appeal response
      case:
        attachments_heading: Response` it should work

Reply:
Thanks I didn't realise you could set a default.

Reply:
I didn;t realise you could use a default. the only problem is that you then have to add the default key to the ignore list